R/pkg/R/WindowSpec.R

```diff
@@ -82,16 +82,18 @@ setMethod("partitionBy",
             }
           })
 
-#' orderBy
+#' Ordering Columns in a WindowSpec
 #'
 #' Defines the ordering columns in a WindowSpec.
-#'
 #' @param x a WindowSpec
-#' @return a WindowSpec
-#' @rdname arrange
+#' @param col a character or Column object indicating an ordering column
+#' @param ... additional sorting fields
+#' @return A WindowSpec
```

Comment:
It seems that the docs I saw from `base` package lower case the arguments and capitalize the first letter of return value part.